src/index.html

```diff
@@ -2,13 +2,60 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0,
+      maximum-scale=1.0, minimum-scale=1.0"
+    >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./reset.css">np
```

Comment:
![image](https://user-images.githubusercontent.com/37161315/137104610-442fe164-1f5a-4d40-897d-bbe39a4ecfda.png)
```suggestion
    <link rel="stylesheet" href="./reset.css">
```